docs: add attribution and rolebinding explanations (#3)
hacbs-release/hacbs-demos: mvp/README.md

`oc apply -f release/dev-workspace/release_plan.yaml -n dev-release-team-tenant`

Standing Attribution for ReleasePlan

In order for the release process to work successfully, there must be a username to attribute the releasing of the

> particular images to. This document discusses what is called the automated usecase, meaning the release resource is

Reviewer: ```suggestion
particular images to. This document discusses what is called the automated use case, meaning the Release resource is
```

Author: fixed

> created by the integration service when the component builds complete, instead of the release resource being

Reviewer: ```suggestion
created by the integration service when the component builds complete, instead of the Release resource being
```

Author: fixed

> created manually by a user. For the automated usecase, the attribution username is retrieved from the ReleasePlan. If

Reviewer: ```suggestion
created manually by a user. For the automated use case, the attribution username is retrieved from the ReleasePlan. If
```

Author: fixed

the ReleasePlan has `release.appstudio.openshift.io/standing-attribution: 'true'`, which this one does, the user who
creates or most recently modifies the ReleasePlan is set as the author of it and that value is used by the release

> pipelineruns. Because the standing-attribution label is already set here, there is nothing to do, but it is something

Reviewer: ```suggestion
PipelineRuns. Because the standing-attribution label is already set here, there is nothing to do, but it is something
```

Author: fixed

to be mindful of.

Create RoleBinding for Release PipelineRuns

The release pipelines require access to certain custom resources in both the dev and managed workspaces. To provide this access, a

> rolebinding must be created for each workspace. The managed workspace one will be created later by the bootstrap scripts, but the

Reviewer: ```suggestion
RoleBinding must be created for each workspace. The managed workspace one will be created later by the bootstrap scripts, but the
```

Author: fixed
